loops-and-arrays.js

```diff
@@ -3,35 +3,38 @@
 // Schrijf een script dat iedere naam in onderstaande array vervangt door een koosnaampje, door er -"je" achter te plakken.
 // Zorg ervoor dat dit ook zou werken als de array wel 100 of 200 namen zou bevatten!
 // ==========================================
-
 const names = ["Henk", "Piet", "Fred", "Joop"];
 
 // Verwachtte uitkomsten:
 // Vóór het script zie je de originele waardes:
 // console.log(names) geeft: ["Henk", "Piet", "Fred", "Joop"]
 // Na jouw script zie je de aangepaste waardes:
 // console.log(names) geeft: ["Henkje", "Pietje", "Fredje", "Joopje"]
-
-
-
+for (let i = 0; i < names.length; i++)
+    names[i] = names[i] + "je";
+console.log(names);
 
 
 // ==========================================
 // Opdracht 2
 // Schrijf een script dat ieder EVEN getal in onderstaande array met 2 vermenigvuldigd, en ieder ONEVEN getal met 3
 // Zorg ervoor dat dit ook zou werken als de array wel 100 of 200 getallen zou bevatten!
 // ==========================================
-
 const numbers = [2, 4, 5, 29, 38];
 
 // Verwachtte uitkomsten:
 // Vóór het script zie je de originele waardes:
 // console.log(numbers) geeft: [2, 4, 5, 29, 38];
 // Na jouw script zie je de aangepaste waardes:
 // console.log(numbers) geeft: [ 4, 8, 15, 87, 76 ];
-
-
-
+for (i = 0; i < numbers.length; i++)
+    if (numbers[i] % 2 === 0) {
+        numbers[i] = numbers[i] * 2;
+    }
+else if (numbers[i] % 2 !== 0) {
+    numbers[i] = numbers[i] * 3;
+}
```

Comment:
Door je al weet dat je in de else zit, moet je expressie altijd waar zijn (en kun je die dus weglaten)
```suggestion
else {
    numbers[i] = numbers[i] * 3;
}
```